BART_evaluations.txt

```diff
@@ -0,0 +1,3 @@
+ROUGE1 score: AggregateScore(low=Score(precision=0.28437878595922683, recall=0.7859388888279921, fmeasure=0.40167933124097793), mid=Score(precision=0.3302329966344789, recall=0.839176344461914, fmeasure=0.4455972190531664), high=Score(precision=0.38074499496583225, recall=0.8866876333778365, fmeasure=0.4922773836004021))
```

Comment:
Could we throw the evaluation and sentence text files into a subfolder in data? Let's try and keep the top level of the repo clean.

Reply:
Good idea, done